src/primitives/transaction.h

```diff
@@ -61,13 +61,39 @@ class CTxIn
     CScript scriptSig;
     uint32_t nSequence;
 
+    /* Setting nSequence to this value for every input in a transaction
+     * disables nLockTime. */
+    static const uint32_t SEQUENCE_FINAL = 0xffffffff;
```

Comment:
This comment has nothing to do with BIP 68. Setting all nSequence to max into disables nLockTime for all tx versions, today.